aspnetcore/security/authorization/secure-data/samples/final15/Authorization/ContactRoleAuthorizationHandler.cs

```diff
@@ -16,7 +16,7 @@ public class ContactRoleAuthorizationHandler
                 return Task.FromResult(0);
             }
 
-            if (context.User.IsInRole(requirement.Name))
+            if (context.User.IsInRole(Constants.OperationRolePrefix + requirement.Name))
```

Comment:
Thanks @blowdart.
That makes sense to me. 
Should the ContactRoleAuthorizationHandler handle both Roles, succeed if in ContactAdministrators else if the user is in Users role check that they are also the owner of the contact? Or should we split that into two handlers like ContactAdminAuthoriationHandler and ContactUserAuthorizationHandler?

Currently we also have ContactIsOwnerAuthorizationHandler which succeeds if the user is the contact owner but does not do any role check. Should we remove that one or change it to include the check for the "Users" role? Maybe the name should be changed to ContactUsersAuthorizationHandler?

Then I guess users created through the UI Registration will only be able to view since they will not have nay roles.